wren-ai-service/Justfile

```
@@ -1,5 +1,37 @@
+up:
+	just prepare-wren-engine
+	docker compose -f ./tools/dev/docker-compose-dev.yaml --env-file ./tools/dev/.env.example up -d
+
+down:
+	docker compose -f ./tools/dev/docker-compose-dev.yaml --env-file ./tools/dev/.env.example down
+
+start:
+	poetry run python -m src.__main__
+
+curate_eval_data:
+	poetry run streamlit run eval/data_curation/app.py
+
 predict file:
     @python3 -u eval/prediction.py --file {{file}}
 
 eval file:
-    @python3 -u eval/evaluation.py --file {{file}}
\ No newline at end of file
+    @python3 -u eval/evaluation.py --file {{file}}
+
+demo:
+	poetry run streamlit run demo/app.py
+
+test:
+	just up
+	poetry run pytest -s $(args)
+	just down
```

Alternative, refer to https://github.com/casey/just?tab=readme-ov-file#running-recipes-at-the-end-of-a-recipe.
```suggestion
test: up && down
	poetry run pytest -s $(args)
```